### DISCUSSION TITLE
Pop up cascade during compiling of test file

### DISCUSSION THREAD

**Discussion by boonebeausoleil** [idaholab/moose#20837] [Q&A Getting Started] [answered]
It had been a long while since I ran anything and had to do a complete re-install of Moose, Mamba, etc. I finished the Moose install and went to run the compiling test and at some point started getting a giant cascade of pop ups asking for permission. I would love to tell you what stage in the compiling it occurred but I couldn't get through them to see. I eventually had to do a hard reboot because I couldn't do anything else. Any thoughts?

<img width="1379" alt="Screen Shot 2022-04-21 at 10 07 55 AM" src="https://user-images.githubusercontent.com/65725166/164508068-fb5514a1-2e5b-4c82-981a-8f84e3e72f9a.png">

**Comment by GiudGiud** [comment, marked as answer]
see https://github.com/idaholab/moose/discussions/16876
